payjoin-ffi/src/test_utils.rs

```diff
@@ -130,7 +130,11 @@ impl TestServices {
 
     pub fn directory_url(&self) -> Url {
         let runtime = RUNTIME.lock().expect("Lock should not be poisoned");
-        runtime.block_on(async { self.0.lock().await.directory_url().into() })
+        runtime.block_on(async {
+            payjoin::Url::try_from(self.0.lock().await.directory_url().to_string())
+                .expect("Could not parse Url")
+                .into()
```

Comment:
Wouldn't it make sense to convert `url::Url` to a `payjoin::Url` inside the `payjoin-test-utils` definition? then this conversion wouldn't be a try-from `url::Url` -> `str` -> `payjoin_ffi::Url` but could just be a plain from conversion that wraps `payjoin::Url(url::Url)` directly?